### DISCUSSION TITLE
Key mapping possible to bind to swipe?

### DISCUSSION THREAD

**Discussion by puppymomo02** [PlayCover/PlayCover#1043] [Q&A]
Hello, I am playing a game where swiping bottom left corner to right is needed (not from center).  I can not find a way to do this. Is it possible?

basically on an iphone, it would be: touch bottom left corner > swipe right
on PlayCover, it is: click bottom left corner > drag to right > release mouse

**Comment by rakuzen25** [comment]
Is there any response to this? I'm looking for the same thing…

**Comment by limingxinleo** [comment]
I need this
